tutorials/TR/README.md

```diff
@@ -0,0 +1,93 @@
+# 🤗 Eğitim Araç Seti
+
+<aside>
+
+💡 **Hoş Geldiniz!**
+
+Üniversite eğitmenlerinin kolayca laboratuvar, ödev veya ders hazırlamak için kullanabilecekleri bir araç seti oluşturduk. İçerik, mevcut müfredata kolayca dahil edilebilecek ve kendi kendine yetebilecek şekilde tasarlanmıştır. Bu içerik **ücretsizdir** ve yaygın olarak bilinen Açık Kaynak teknolojilerini (`transformerlar`, `gradio`, vb.) kullanır.
+
+Alternatif olarak, Hugging Face ekibinden birinden [ML demo.cratization turu](https://www.notion.so/ML-Demo-cratization-tour-with-66847a294abd4e9785e85663f5239652) girişimi aracılığıyla sınıfınız için eğitimleri yapmasını isteyebilirsiniz!
+
+Eğitimlerin yanı sıra, makine öğreniminde daha ileri gidebileceğiniz veya kurs içeriği tasarlamaya yardımcı olabilecek diğer kaynakları da paylaşıyoruz.
+
+</aside>
+
+## **Eğitim Kataloğumuz**
+
+### 1️⃣ Hugging Face Hub'da Bir Tur
+
+> Bu eğitimde şunları elde edersiniz:
+>
+> - Hub'da paylaşılan 30.000'den fazla modeli keşfedin.
+> - Kendi göreviniz için doğru modeli ve veri kümelerini bulmanın etkili yollarını öğrenin.
+> - Makine öğrenimi iş akışlarınıza nasıl katkıda bulunacağınızı ve işbirliği içinde nasıl çalışacağınızı öğrenin
+>
+> **_Süre: 20-40 dakika_**
+>
+> 👉 [eğitime erişmek için buraya tıklayın](https://www.notion.so/Workshop-A-Tour-through-the-Hugging-Face-Hub-2098e4bae9ba4288857e85c87ff1c851) veya 👩‍🏫 [ders sunumları](https://docs.google.com/presentation/d/1zQqpFTcpNLV7haj2Inw2qKHq8DjfZEaiObW1ZkLvPWM/edit?usp=sharing).
+
+### 2️⃣ Gradio ve Hugging Face ile Makine Öğrenimi Demoları Oluşturun ve Barındırın
+
+> Bu eğitimde şunları elde edersiniz:
+>
+> - Topluluk tarafından oluşturulan ML demolarını keşfedin.
+> - `gradio` kütüphanesini kullanarak Python'da makine öğrenimi modeliniz için hızlı bir demo oluşturun
+> - Hugging Face Spaces ile demoları ücretsiz olarak barındırın
+> - Sınıfınız veya konferansınız için demonuzu Hugging Face kuruluşuna ekleyin
+>
+> **_Süre: 20-40 dakika_**
+>
+> 👉 [eğitime erişmek için buraya tıklayın](https://colab.research.google.com/github/huggingface/education-toolkit/blob/main/tutorials/TR/02_ml-demos-with-gradio.ipynb) veya 👩‍🏫 [ders sunumları](https://docs.google.com/presentation/d/14EU_xjtINXtpidWLnUvfcEpmxN46ORS-PLpwfUf8C1I/edit?usp=sharing).
+
+### 3️⃣ Transformerlara Giriş
+
+> Bu eğitimde şunları elde edersiniz:
+>
+> - Transformer sinir ağları, doğal dil işleme ve ötesinde çok çeşitli görevlerin üstesinden gelmek için kullanılabilir.
+> - Transfer öğrenimi, kişinin transformerları belirli görevlere uyarlamasını sağlar.
```

Comment:
let's use transformers instead of transformerlar

Reply:
I think we could instead say transformer'ları given it doesn't make sense to say transformers (it's completely English)